NAMESPACE

```diff
@@ -1 +1,7 @@
-exportPattern("^[[:alpha:]]+")
+# Generated by roxygen2: do not edit by hand
```

Comment:
hi @RobertMyles , to be honest I am not a great expert of roxygen. could you explain me a bit about this lines of code? How did you generated them? thanks!